{"name": "test_calculator[2-4-8]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 6 == 8\n +  where 6 = <function add_sum at 0x00000196EAF37C40>(2, 4)\n +    where <function add_sum at 0x00000196EAF37C40> = calculator_08.add_sum", "trace": "x = 2, y = 4, expected = 8\n\n    @pytest.mark.parametrize(\"x, y, expected\", data)\n    def test_calculator(x , y, expected):\n>       assert calculator_08.add_sum(x, y) == expected\nE       assert 6 == 8\nE        +  where 6 = <function add_sum at 0x00000196EAF37C40>(2, 4)\nE        +    where <function add_sum at 0x00000196EAF37C40> = calculator_08.add_sum\n\ntests\\test_calculator_08.py:7: AssertionError"}, "parameters": [{"name": "x", "value": "2"}, {"name": "y", "value": "4"}, {"name": "expected", "value": "8"}], "start": 1736980148319, "stop": 1736980148319, "uuid": "0b08c81b-7d30-48af-9f43-07bb70074bd5", "historyId": "efe9f3e4581a21f7514d5f4d616d6895", "testCaseId": "b1df44ec2cf6ca650d87e936d02c4be9", "fullName": "tests.test_calculator_08#test_calculator", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_calculator_08"}, {"name": "host", "value": "Mali_Vaknin"}, {"name": "thread", "value": "4676-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_calculator_08"}]}